src/openzaak/templates/admin/catalogi/import_zaaktype.html

```diff
@@ -0,0 +1,13 @@
+{% extends "admin/base_site.html" %}
+{% load i18n %}
+
+{% block title %} {% trans "Import ZaakType" %} {{ block.super }} {% endblock %}
+
+{% block content %}
```

Comment:
I gave a proper example after redoing some layout for catalogus, can you add it here too,